erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.js

```diff
@@ -21,7 +21,45 @@ frappe.ui.form.on("Payment Reconciliation Payment", {
 				doc: frm.doc,
 				method: 'get_difference_amount',
 				args: {
-					child_row: row
+					child_row: row,
+					source_table : 'payments'
+				},
+				callback: function(r, rt) {
+					if(r.message) {
+						frappe.model.set_value(cdt, cdn,
+							"difference_amount", r.message);
+					}
+				}
+			});
+		}
+	}
+});
+
+frappe.ui.form.on("Payment Reconciliation Invoice",{
```

Comment:
```suggestion
frappe.ui.form.on("Payment Reconciliation Invoice", {
```
Add single space after comma everywhere